plutus-tx/src/PlutusTx/Blueprint/Schema.hs

```diff
@@ -136,22 +140,11 @@ instance ToJSON (Schema referencedTypes) where
     dataType info ty = requiredField "dataType" ty (infoFields info)
 
     infoFields :: SchemaInfo -> Aeson.Object
-    infoFields MkSchemaInfo{title, description, comment} =
+    infoFields (Set.toList -> info) =
       KeyMap.empty
-        & optionalField "title" title
-        & optionalField "description" description
-        & optionalField "$comment" comment
-
--- | Additional information optionally attached to any datatype schema definition.
-data SchemaInfo = MkSchemaInfo
```

Comment:
I'd have been tempted to keep this and write a `Set SchemaAnn -> SchemaInfo` function. In particular, it seems to me that that function should fail if you have duplicate annotations, whereas currently it will arbitrarily pick one, which seems bad.